tests/integration/security/main_test.go

```diff
@@ -55,6 +55,8 @@ meshConfig:
   accessLogEncoding: JSON
   accessLogFile: /dev/stdout
   defaultConfig:
+    image:
```

Comment:
is this intended...?

Reply:
yeah, I want the distroless tests to use this API.
I will make some more test changes to make the other tests pass now.

Reply:
Would it be more appropriate to have the test that sets `VARIANT=distroless` to set this (for all tests)? 

Since right now I think this means that all usages of ./test/integration/security use distroless

Reply:
btw - I have short term plans to make distroless run more than just ./test/integration/security

Reply:
Now it will only run distroless if VARIANT var is present. I did not want to spend more time on disentangling it.